vsce/client/src/util/spawnVersionCompareJson.ts

```diff
@@ -72,7 +72,7 @@ export default (
 						process.stderr.readableLength
 					)
 				);
-				reject(null);
+				return reject(null);
```

Comment:
`return` is important here so we don't fall through to the `resolve` below.

Reply:
That makes sense.